crates/nostr-postgresdb/examples/postgres-relay.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2025 Protom
+// Distributed under the MIT software license
+
+use std::time::Duration;
+
+use nostr_database::prelude::*;
+use nostr_postgresdb::NostrPostgres;
+use nostr_relay_builder::prelude::*;
+
+// Your database URL
+const DB_URL: &str = "postgres://postgres:password@localhost:5432";
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    tracing_subscriber::fmt::init();
+
+    // This will programatically run pending db migrations
+    nostr_postgresdb::run_migrations(DB_URL)?;
+
+    // Create a conncetion pool
+    let pool = nostr_postgresdb::postgres_connection_pool(DB_URL).await?;
+
+    // Create a nostr db instance
+    let db: NostrPostgres = pool.into();
```

Comment:
IMO the migrations should run under the hood, or during the construction of the db instance or later, before the first interaction with the db connection. What do your think?

Reply:
IDK, people who add this to an existing DB might want to run the migration manually (eg with diesel cli). I can add a constructor variant that runs them automatically though.